chargeOclock/Source/Models/Client.swift

```diff
@@ -51,4 +51,9 @@ extension Client {
 		}
 	}
 
+	func remove(completion: @escaping (Error?) -> Void) {
```

Comment:
rename to `delete` (we're not removing a client from a collection, but actually deleting it from the database)